src/vcpkg/commands.set-installed.cpp

```diff
@@ -277,6 +278,23 @@ namespace vcpkg
             }
         }
 
+        if (auto manifest = paths.get_manifest().get())
+        {
+            Json::Object manifest_info;
+            manifest_info.insert("manifest_path", Json::Value::string(manifest->path));
+
+            if (const auto installed_paths = paths.maybe_installed().get())
+            {
+                const auto json_file_path = installed_paths->vcpkg_dir() / "manifest_info.json";
```

Comment:
```suggestion
                const auto json_file_path = installed_paths->vcpkg_dir() / "manifest-info.json";
```

I think we're always doing dashes for anything new.

Also this name should go in `contractual-constants.h`.

Reply:
Hmmmm though vcpkg_abi_info.txt has underscores, so using _ would be consistent with the other file we put here. I guess you can take or leave this suggestion then :). (But -> contractual-constants.h is still a thing)